packages/mobile/src/app/saga.ts

```diff
@@ -130,8 +141,30 @@ export function handleDeepLink(deepLink: string) {
   // Other deep link handlers can go here later
 }
 
+export function* navigatePinProtected(action: NavigatePinProtected) {
+  const zeroSyncMode = yield select(zeroSyncSelector)
+  try {
+    if (!zeroSyncMode) {
+      const pincode = yield call(getPincode, false)
+      yield put(startPinVerification())
+      const account = yield call(getConnectedAccount)
```

Comment:
This line is potentially problematic. It means that the navigation won't proceed until the app is done syncing, which seems like an unnecessary requirement for this. Recommend using `getAccount` instead